src/python_pachyderm/util.py

```diff
@@ -69,7 +70,7 @@ def put_files(client, source_path, commit, dest_path, **kwargs):
 
 
 def create_python_pipeline(
-    client,
+    client: Client,
```

Comment:
Just curious, since I don't understand python type annotations that well: does this add functionality or is it just a convenience for users?

Reply:
Python type hints don't actually enforce types in the same way that statically typed languages do. But this helps IDEs and linters. In this example, the IDE now understands that client is of type `Client`, and can highlight its members, as well provide auto suggestions.

Type hints also help with documenting code. Currently, we specify the arg type in our docstring, but this is adhoc, and doc generators are not necessarily going to pick this up automatically. Python type hints standardizes this, so doc generators can parse them more easily.

If we decide to add type hints throughout our project, it would be quite a massive undertaking though. I think moving forward, we can try to add type hints incrementally, and definitely for new functions.